MIGRATION_GUIDE.md

```diff
@@ -52,8 +52,11 @@ Changes are grouped as follows:
 - `client.data_modeling.instances.aggregate` the parameters `instance_type` and `group_by` has swapped order.
 - The return type of `client.data_modeling.instances.aggregate` has changed from `InstanceAggregationResultList` to
   a more specific value `AggregatedNumberedValue | list[AggregatedNumberedValue] | InstanceAggregationResultList` depending on the `aggregates` and `group_by` parameters.
-- The `client.sequences.data.retrieve` method has changed signature:
-  The parameter `columns_external_id` is renamed `columns`. This is to better match the API and have a consistent overload implementation.
+- The `client.sequences.data` methods `.retrieve`, `.retrieve_last_row`, `.insert`  method has changed signature:
```

Comment:
Since `retrieve_last_row` is "new", it reads a bit odd that it has changed its signature.